hologram_dashboard/fetcher/fetcher.cc

```diff
@@ -0,0 +1,21 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+#include "fetcher.h"
+
+int main() {
+    std::cout << "Hi\n";
+    return 0;
+}
```

Comment:
You need a separate file to put main(), like fetcher_main.cc.

In fetcher.cc, we should only include functions.